Makefile

```diff
@@ -54,7 +54,7 @@ init: ##@0 global cleanup/install/bootstrap
 
 fmt: ##@0 global format code using prettier (js, css, md)
 	@./node_modules/.bin/prettier --color --write \
-		"packages/*/{src,stories,tests}/**/*.{js,ts}" \
+		"packages/*/{src,stories,tests}/**/*.{js,ts,tsx}" \
```

Comment:
Did you want to update `fmt-check` below to to have `tsx` extension as well?

Reply:
Yes, added, thx.